scripts/interactive_tester.py

```diff
@@ -55,12 +57,18 @@ def get_parser():
                         - mpl
                         - pandas
                         - seaborn
+
+                    It also allows to specify a example custom folder path, like:
```

Comment:
Right.. I added a bad description. Lines 48-49 were already there and meant you can only selected from those "pre-built" locations. I intended to say that now you can also specify a custom valid path. 
I'll improve the docstring..